manifests/t/twpayne/chezmoi/2.34.3/twpayne.chezmoi.installer.yaml

```diff
@@ -0,0 +1,25 @@
+# Created with WinGet Releaser v2 using Komac v1.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: twpayne.chezmoi
+PackageVersion: 2.34.3
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: chezmoi.exe
+  PortableCommandAlias: chezmoi
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/twpayne/chezmoi/releases/download/v2.34.3/chezmoi_2.34.3_windows_arm64.zip

```

Comment:
```suggestion
  InstallerUrl: https://github.com/twpayne/chezmoi/releases/download/v2.34.3/chezmoi_2.34.3_windows_i386.zip
```

Reply:
Thanks for the suggestion. The SHA256 sums also need to be updated. I've done this.